<ui version="4.0" >
 <class>WindowBase</class>
 <widget class="QMainWindow" name="WindowBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form1</string>
  </property>
  <widget class="QWidget" name="widget" >
   <layout class="QVBoxLayout" >
    <property name="margin" >
     <number>11</number>
    </property>
    <property name="spacing" >
     <number>6</number>
    </property>
    <item>
     <layout class="QHBoxLayout" >
      <property name="margin" >
       <number>0</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <widget class="QToolButton" name="saveFormula" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
        <property name="text" >
         <string/>
        </property>
        <property name="icon" >
         <iconset resource="icons.qrc" >:/icons/endturn.svg</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="cancelFormula" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
        <property name="text" >
         <string/>
        </property>
        <property name="icon" >
         <iconset resource="icons.qrc" >:/icons/editdelete.svg</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="cellName" >
        <property name="minimumSize" >
         <size>
          <width>64</width>
          <height>0</height>
         </size>
        </property>
        <property name="text" >
         <string/>
        </property>
        <property name="alignment" >
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="wordWrap" >
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="formula" />
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget2" >
      <widget class="QWidget" name="tab" >
       <attribute name="title" >
        <string>Page 1</string>
       </attribute>
       <layout class="QVBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="QTableWidget" name="grid" >
          <property name="editTriggers" >
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="dragDropOverwriteMode" >
           <bool>false</bool>
          </property>
          <property name="alternatingRowColors" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="MenuBar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>29</height>
    </rect>
   </property>
   <widget class="QMenu" name="fileMenu" >
    <property name="title" >
     <string>&amp;File</string>
    </property>
    <addaction name="fileNewAction" />
    <addaction name="fileOpenAction" />
    <addaction name="fileSaveAction" />
    <addaction name="fileSaveAsAction" />
    <addaction name="separator" />
    <addaction name="filePrintAction" />
    <addaction name="separator" />
    <addaction name="fileExitAction" />
   </widget>
   <widget class="QMenu" name="editMenu" >
    <property name="title" >
     <string>&amp;Edit</string>
    </property>
    <addaction name="editUndoAction" />
    <addaction name="editRedoAction" />
    <addaction name="separator" />
    <addaction name="editCutAction" />
    <addaction name="editCopyAction" />
    <addaction name="editPasteAction" />
    <addaction name="separator" />
    <addaction name="editFindAction" />
   </widget>
   <widget class="QMenu" name="helpMenu" >
    <property name="title" >
     <string>&amp;Help</string>
    </property>
    <addaction name="helpContentsAction" />
    <addaction name="separator" />
    <addaction name="helpAboutAction" />
   </widget>
   <addaction name="fileMenu" />
   <addaction name="editMenu" />
   <addaction name="helpMenu" />
  </widget>
  <widget class="QToolBar" name="toolBar" >
   <property name="orientation" >
    <enum>Qt::Horizontal</enum>
   </property>
   <attribute name="toolBarArea" >
    <number>4</number>
   </attribute>
   <addaction name="fileNewAction" />
   <addaction name="fileOpenAction" />
   <addaction name="fileSaveAction" />
   <addaction name="filePrintAction" />
   <addaction name="separator" />
   <addaction name="editUndoAction" />
   <addaction name="editRedoAction" />
   <addaction name="separator" />
   <addaction name="editCutAction" />
   <addaction name="editCopyAction" />
   <addaction name="editPasteAction" />
  </widget>
  <widget class="QStatusBar" name="statusBar" />
  <action name="fileNewAction" >
   <property name="icon" >
    <iconset resource="icons.qrc" >:/icons/filenew.svg</iconset>
   </property>
   <property name="text" >
    <string>&amp;New</string>
   </property>
   <property name="iconText" >
    <string>New</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="fileOpenAction" >
   <property name="icon" >
    <iconset resource="icons.qrc" >:/icons/fileopen.svg</iconset>
   </property>
   <property name="text" >
    <string>&amp;Open...</string>
   </property>
   <property name="iconText" >
    <string>Open</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="fileSaveAction" >
   <property name="icon" >
    <iconset resource="icons.qrc" >:/icons/filesave.svg</iconset>
   </property>
   <property name="text" >
    <string>&amp;Save</string>
   </property>
   <property name="iconText" >
    <string>Save</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="fileSaveAsAction" >
   <property name="icon" >
    <iconset resource="icons.qrc" >:/icons/filesaveas.svg</iconset>
   </property>
   <property name="text" >
    <string>Save &amp;As...</string>
   </property>
   <property name="iconText" >
    <string>Save As</string>
   </property>
   <property name="shortcut" >
    <string/>
   </property>
  </action>
  <action name="filePrintAction" >
   <property name="icon" >
    <iconset resource="icons.qrc" >:/icons/fileprint.svg</iconset>
   </property>
   <property name="text" >
    <string>&amp;Print...</string>
   </property>
   <property name="iconText" >
    <string>Print</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="fileExitAction" >
   <property name="icon" >
    <iconset resource="icons.qrc" >:/icons/exit.svg</iconset>
   </property>
   <property name="text" >
    <string>E&amp;xit</string>
   </property>
   <property name="iconText" >
    <string>Exit</string>
   </property>
   <property name="shortcut" >
    <string/>
   </property>
  </action>
  <action name="editUndoAction" >
   <property name="icon" >
    <iconset resource="icons.qrc" >:/icons/undo.svg</iconset>
   </property>
   <property name="text" >
    <string>&amp;Undo</string>
   </property>
   <property name="iconText" >
    <string>Undo</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="editRedoAction" >
   <property name="icon" >
    <iconset resource="icons.qrc" >:/icons/redo.svg</iconset>
   </property>
   <property name="text" >
    <string>&amp;Redo</string>
   </property>
   <property name="iconText" >
    <string>Redo</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="editCutAction" >
   <property name="icon" >
    <iconset resource="icons.qrc" >:/icons/editcut.svg</iconset>
   </property>
   <property name="text" >
    <string>Cu&amp;t</string>
   </property>
   <property name="iconText" >
    <string>Cut</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="editCopyAction" >
   <property name="icon" >
    <iconset resource="icons.qrc" >:/icons/editcopy.svg</iconset>
   </property>
   <property name="text" >
    <string>&amp;Copy</string>
   </property>
   <property name="iconText" >
    <string>Copy</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="editPasteAction" >
   <property name="icon" >
    <iconset resource="icons.qrc" >:/icons/editpaste.svg</iconset>
   </property>
   <property name="text" >
    <string>&amp;Paste</string>
   </property>
   <property name="iconText" >
    <string>Paste</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="editFindAction" >
   <property name="icon" >
    <iconset resource="icons.qrc" >:/icons/find.svg</iconset>
   </property>
   <property name="text" >
    <string>&amp;Find...</string>
   </property>
   <property name="iconText" >
    <string>Find</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="helpContentsAction" >
   <property name="icon" >
    <iconset resource="icons.qrc" >:/icons/help.svg</iconset>
   </property>
   <property name="text" >
    <string>&amp;Contents...</string>
   </property>
   <property name="iconText" >
    <string>Contents</string>
   </property>
   <property name="shortcut" >
    <string/>
   </property>
  </action>
  <action name="helpAboutAction" >
   <property name="text" >
    <string>&amp;About</string>
   </property>
   <property name="iconText" >
    <string>About</string>
   </property>
   <property name="shortcut" >
    <string/>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources>
  <include location="icons.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>formula</sender>
   <signal>returnPressed()</signal>
   <receiver>saveFormula</receiver>
   <slot>animateClick()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
